### PR TITLE
Add button positions and sizes

### PR DESCRIPTION
**Pull request by TobiasBales**
This allows setting `width:`, `height:`, `top:`, and `left:` for buttons.

This should address the coordinate and size points of #4

The properties also all work independently, so you can set only width and top for example.

Screenshot 2023-02-06 at 11 58 50

Usage:
```ruby
@button = button "Push me", width: 200, height: 50, top: 109, left: 132
```

You can try it by running `bundle exec ruby examples/button_with_position_and_size.rb` from the repo

### CODE REVIEW

File: examples/button_with_position_and_size.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require "scarpe"
```
Comment (Schwad) [inline examples/button_with_position_and_size.rb:1]: Thank you for including an example for testing! Really helpful!

File: lib/scarpe/button.rb [hunk 2]
```diff
@@ -17,11 +21,24 @@ def click(&block)
 
     def render
       @app.bind(function_name) do
-        if @block
-          @block.call
-        end
+        @block.call if @block
       end
-      "<button id=#{object_id} onclick='scarpeHandler(#{function_name})'>#{@text}</button>"
+      "<button id=#{object_id} onclick='scarpeHandler(#{function_name})' style='#{style}'>#{@text}</button>"
```
Comment (Schwad) [inline lib/scarpe/button.rb:26]:
```suggestion
      puts "<button id=#{object_id} onclick='scarpeHandler(#{function_name})' style='#{style}'>#{@text}</button>"
      "<button id=#{object_id} onclick='scarpeHandler(#{function_name})' style='#{style}'>#{@text}</button>"
```

I'd like to keep up Marco's pattern of logging the resulting HTML to the terminal, is that okay? 

I know it looks a bit funky 'puts'ing something and then returning it. But right now I don't see a sneaky way around it. Unless we create a parent private method `render` with a `super` or something to do a bit of cleverness. But I think we can be repetitive for now?

**Review by Schwad — APPROVED** [review]
I trialed it out locally, looks great! Thank you!

Some examples of me going crazy with the button:

Screenshot 2023-02-06 at 13 20 49
Screenshot 2023-02-06 at 13 21 57

I have a one-liner suggestion for logging the HTML but other than that happy to ship this

File: lib/scarpe/button.rb (hunk 2 above)
Comment (TobiasBales) [reply lib/scarpe/button.rb:26]: maybe just assign it to a variable? ;)

Comment (Schwad) [reply lib/scarpe/button.rb:26]: Noah had a good point about debug functionality going forward, which may get more elegant, so I've merged! :D

Comment (Schwad) [reply lib/scarpe/button.rb:26]: aye it'd be a shorter

```ruby
puts @some_stuff
@some_stuff
```

but I think we'll do better

**Pull request merged**